code/lang/TTTLanguage.cs

```diff
@@ -62,5 +69,47 @@ public static Language GetLanguageByName(string name)
 
             return lang;
         }
+
+        public static void UpdateLanguage(Language language)
+        {
+            TTTLanguage.ActiveLanguage = language;
+
+            if (Host.IsClient)
+            {
+                UI.TranslationLabel.UpdateLanguage(language);
+            }
+        }
+    }
+}
+
+namespace TTTReborn.Player
+{
+    using Globalization;
+
+    public partial class TTTPlayer
+    {
+        [ClientCmd("ttt_language")]
+        public static void ChangeLanguage(string name = null)
+        {
+            if (name is null)
+            {
+                Log.Info($"Your current language is set to '{TTTLanguage.ActiveLanguage.Data.Name}' ('{TTTLanguage.ActiveLanguage.Data.Code}').");
+
+                return;
+            }
```

Comment:
What's the purpose of providing a default arg of `null` here, if someone calls the function with no args they just get a print statement? Seems a bit off since the name of the function is `ChangeLanguage`

Reply:
Yep, to use it like a convar